app/controllers/api/v1/topics_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module V1
+    class TopicsController < Api::V1::ApiController
+      def index
+        @topics = Topic.get_all
```

Comment:
Do we need to use this scope?
```suggestion
        @topics = Topic.all
```